added clarity to set-up service account in GCP (#7250)
segmentio/segment-docs: src/connections/storage/catalog/bigquery/index.md

### Create a service account for Segment

To create a service account for Segment: 

> 1. From the Navigation panel on the left in the Google Developer Console, select **IAM & admin** > **Service accounts**.

Reviewer: ```suggestion
1. From the Google Developer Console, select the Navigation panel and navigate to **IAM & admin** > **Service accounts**.
```
